soc/infineon/cat1b/cyw20829/mcuboot.cmake

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# This file includes extra build system logic that is enabled when
+# CONFIG_BOOTLOADER_MCUBOOT=y.
+#
+# It builds signed binaries using cysecuretools as a post-processing step
+# after zephyr/zephyr.elf is created in the build directory.
```

Comment:
can you explain why this is needed?

Reply:
HI @nordicjm,
MCUboot support for CYW20829 device, has couple specifics:
- It supported only by Cypress branch of MCUboot https://github.com/mcu-tools/mcuboot/tree/cypress
>> we will provide documentation in board documentation how to build Cypress MCUboot bootloader (not Zephyr app) and operate with  with Zephyr (CONFIG_BOOTLOADER_MCUBOOT=y)

- In CYW20829 image,  encryption has some customization so we are using cysecuretools instead imagetool to sign/encrypt image, so [soc/infineon/cat1b/cyw20829/mcuboot.cmake](https://github.com/zephyrproject-rtos/zephyr/pull/81121/files#diff-601b7d4376d291f2ab0439b8a12eddb2be585adf8f023378ea7ede6f97cc292e) implements sign/encrypt by cysecuretools.